packages/webapp/pages/join/organization.tsx

```diff
@@ -0,0 +1,270 @@
+          <ProfileImageLink user={member} className="shrink-0" />
+          <Typography
+            type={TypographyType.Body}
+            color={TypographyColor.Tertiary}
+            className="max-w-full shrink"
+          >
+            <Link href={member.permalink} passHref>
+              <a>
+                <span className="font-bold text-text-primary">
+                  {member.name}
+                </span>{' '}
+                (@{member.username})
+              </a>
+            </Link>{' '}
+            has invited you to join their organization
+          </Typography>
+        </div>
+
+        <Typography
+          type={TypographyType.Callout}
+          color={TypographyColor.Tertiary}
+        >
+          Organizations let your team unlock the full power of daily.dev Plus
+          together including premium features, shared learning, and streamlined
+          user management, all in one place.
+        </Typography>
+
+        <div className="flex items-center gap-4 rounded-16 border border-brand-default p-4">
+          <Image
+            className="size-12 rounded-full object-cover"
+            src={organization.image}
+            alt={`Avatar of ${organization.name}`}
+            type={ImageType.Organization}
+          />
+
+          <div className="flex flex-col">
+            <Typography
+              bold
+              type={TypographyType.Title3}
+              color={TypographyColor.Primary}
+            >
+              {organization.name}
+            </Typography>
+            <Typography
+              type={TypographyType.Body}
+              color={TypographyColor.Tertiary}
+            >
+              {organization.seats} seats available
+            </Typography>
+          </div>
+
+          <Button
+            variant={ButtonVariant.Primary}
+            disabled={organization.seats === 0}
+            className="ml-auto"
+            onClick={onJoinClick}
+            loading={isJoiningOrganization}
+          >
+            Join
+          </Button>
+        </div>
+      </div>
+    </main>
+  );
+};
+
+interface QueryParams {
+  token: string;
+  orgId: string;
+}
+
+export const getServerSideProps: GetServerSideProps<{
+  token: string;
+  organization: Omit<Organization, 'members'>;
+  user: Author;
+}> = async ({ query, res }) => {
+  const params = query as unknown as QueryParams;
+  const { token, orgId } = params;
+
+  if (!token || !orgId) {
+    return {
+      redirect: {
+        destination: '/',
+        permanent: false,
+      },
+    };
+  }
+
+  try {
+    const { getOrganizationByIdAndInviteToken } = await gqlClient.request<{
+      getOrganizationByIdAndInviteToken: {
+        user: Author;
+        organization: Omit<Organization, 'members'>;
+      };
+    }>(GET_ORGANIZATION_BY_ID_AND_INVITE_TOKEN_QUERY, {
+      id: orgId,
+      token,
+    });
+
+    res.setHeader(
+      'Cache-Control',
+      `public, max-age=0, must-revalidate, s-maxage=${24 * 60 * 60}`,
```

Comment:
Should we maybe start putting this number somewhere, just to make it easier to use and understand.

Reply:
Done, 69280b7